packages/varlet-ui/src/code/Code.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div :class="classes(n())">
```

Comment:
If there is only one class, `classes` function is not necessary. `:class="n()"` is better.